frontend/src/pages/modelServing/customServingRuntimes/CustomServingRuntimeAPIProtocolLabel.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Label, Text, TextVariants } from '@patternfly/react-core';
+import { TemplateKind } from '~/k8sTypes';
+import {
+  getAPIProtocolFromTemplate,
+  getEnabledPlatformsFromTemplate,
+} from '~/pages/modelServing/customServingRuntimes/utils';
+import { ServingRuntimeAPIProtocol, ServingRuntimePlatform } from '~/types';
+
+type CustomServingRuntimeAPIProtocolLabelProps = {
+  template: TemplateKind;
+};
+
+const CustomServingRuntimeAPIProtocolLabel: React.FC<CustomServingRuntimeAPIProtocolLabelProps> = ({
+  template,
+}) => {
+  const apiProtocol = getAPIProtocolFromTemplate(template);
+  const isMultiModel = getEnabledPlatformsFromTemplate(template).includes(
+    ServingRuntimePlatform.MULTI,
+  );
+
+  // If it is multi-model, we use REST as default
+  if (!apiProtocol && isMultiModel) {
+    return <Label color="gold">{ServingRuntimeAPIProtocol.REST}</Label>;
+  }
+
+  if (!apiProtocol) {
+    return <Text component={TextVariants.small}>Not defined</Text>;
```

Comment:
Annotations are strings, we try to controll wether the value is one of our enums, but it can happen that it cannot be the case, we need to control wethere the apiProtocol is not either of the values we have in our enum and if not, display Not defined